handler.js

```diff
@@ -140,6 +176,29 @@ async function handleJob(taskId, rcl, message) {
                 //   elapsed: 6650000,     // ms since the start of the process
                 //   timestamp: 864000000  // ms since epoch
                 // }
+                if (process.env.HF_VAR_ENABLE_TRACING === "1") {
```

Comment:
Here (for sending metrics) let's use another guard, e.g. `"If (otelEnabled)"`. Evaluate it at the beginning using the HF_VAR_ENABLE_OTEL env. (That way we evaluate all envs only at the beginning)